lib/routes/apnews/sitemap.ts

```diff
@@ -0,0 +1,98 @@
+import { Route, ViewType } from '@/types';
+import { fetchArticle } from './utils';
+import ofetch from '@/utils/ofetch';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+import asyncPool from 'tiny-async-pool';
+import timezone from '@/utils/timezone';
+const HOME_PAGE = 'https://apnews.com';
+
+export const route: Route = {
+    path: '/sitemap/:route',
+    categories: ['traditional-media'],
+    example: '/apnews/sitemap/ap-sitemap-latest',
+    view: ViewType.Articles,
+    parameters: {
+        route: {
+            description: 'Route for sitemap, excluding the `.xml` extension',
+        },
+    },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['apnews.com/'],
+        },
+    ],
+    name: 'News',
+    maintainers: ['zoenglinghou', 'mjysci', 'TonyRL'],
```

Comment:
Wrong route name. Route name should be unique within each namespace.
Wrong maintainers.